src/node/constant_source.rs

```diff
@@ -186,7 +186,8 @@ impl AudioProcessor for ConstantSourceRenderer {
             current_time += dt;
         }
 
-        true
+        // tail_time false when output has ended this quantum
+        stop_time > next_block_time
```

Comment:
Remembered that we had to output one channel of silence before returning false, but can't find it in the spec actually... I think the check line 165 makes no sense anymore then?

Reply:
handled in e088c95